### ISSUE TITLE
centroid and centered instance models have same outputs.run_name which results in overwritten models

### ISSUE THREAD

**Issue by smidm** [murthylab/sleap#415]
I generated training configuration in `sleap-label`:

![Screenshot from 2020-11-02 12-13-30](https://user-images.githubusercontent.com/820292/97862240-484d4e80-1d05-11eb-8d5b-7a062f223af9.png)

The [training configuration files](https://github.com/murthylab/sleap/files/5474539/test_run.zip) for both models `centered_instance.json` and `centroid.json` contain the same `outputs.run_name`. When the training is run with `train-script.sh` the later model overwrites the former. I managed to generate correct configurations before, but now it's failing repeatedly.

Due to this issue one model is missing and the inference fails with:
```
Traceback (most recent call last):                                                                
  File "/home.dokt/smidm/local/miniconda3/envs/sleap_env/bin/sleap-track", line 4, in <module>    
    import re                                                                                     
  File "/home.dokt/smidm/local/miniconda3/envs/sleap_env/lib/python3.7/site-packages/sleap/nn/inference.py", line 1274, in main
    model_paths_by_head, labels_path=args.labels, policy_args=policy_args                         
  File "/home.dokt/smidm/local/miniconda3/envs/sleap_env/lib/python3.7/site-packages/sleap/nn/inference.py", line 1206, in make_predictor_from_models
    f"Unable to run inference with {list(trained_model_paths.keys())} heads."                     
ValueError: Unable to run inference with ['centered_instance'] heads.    
```

**Comment by talmo** [comment]
Hi @smidm,

Do you only get a single model folder as a result of running the training script?

Assuming yes -- could you try setting the `run_name_prefix` in the config? This is in the outputs section, for example:

```
    "outputs": {
        "save_outputs": true,
        "run_name": "201102_121410",
        "run_name_prefix": "centroid",
```

That should prevent them from being overwritten (you can also just change the run name directly).

Let me know if that works and I'll look into why the generated files don't get renamed appropriately.

**Comment by smidm** [comment]
I already trained successfully with `run_name` fixed to different values for both models.

**Comment by talmo** [comment]
Now implemented in [SLEAP v1.0.10a7](https://github.com/murthylab/sleap/releases/tag/v1.0.10a7) and will be in the next release (v1.0.10).
